code/web/src/modules/survey/api/actions.js

```diff
@@ -32,7 +32,7 @@ export const setSurveyType = crateId => (
           crateId
         })
       })
-      .catch(error => console.error(error))
+      .catch(error => console.error('There\'s an error'))
```

Comment:
This'll get tweaked, was originally implemented for testing

Reply:
Sounds good